spec.bs

```diff
@@ -2280,6 +2268,27 @@ To mend the intended distinction between [=top-level traversables=] and [=fenced
 Note: With these new definitions, a [=top-level traversable=] is essentially "unfenced" as described
 in the [[#nested-traversables-intro]].
 
+The algorithm as it exists today is a "fenced" algorithm, which will be re-introduced as a "fenced"
+variant as follows:
+
+<div algorithm>
+  A <dfn>fenced top-level traversable</dfn> is a [=traversable navigable=] with a null
+  [=navigable/parent=].
+</div>
+
+<div algorithm>
```

Comment:
Also if we're going to do this I think we should just say we're changing the HTML algorithm's name. No need to copy its text exactly, right?